storybook/src/docs/getting-started-developers.docs.mdx

```diff
@@ -2,7 +2,7 @@
 
 import { Meta } from "@storybook/blocks";
 
-<Meta title="Docs/Getting started/Developers" />
+<Meta title="Docs/Developer Guide/Getting started" />
```

Comment:
Should we rename the files for these types of changes? So this would become `developer-guide-getting-started`? Or, if we create the same folder structure: `getting-started` in the 'Developer guide' folder?

Reply:
Yes, with directories.

Reply:
Done.